### PR TITLE
Add toggle for scroll wheel

### PR DESCRIPTION
**Pull request by ytai-chn**
In Genshin Impact, the scroll wheel may cause some bugs:
1. If you use the scroll wheel to the maximum, it will trigger an unexpected camera lock. As the video says, the character moves but the camera locks.
2. When playing games with the touchpad, sometimes I will be stuck in a zoom-only perspective, and at this time, you can no longer rotate the camera but only zoom-in & zoom-out.
After turning off the wheel mapping, the above bugs do not reappear, so I added a toggle to control the scroll wheel mapping. Another PR in playcover repo: https://github.com/PlayCover/PlayCover/pull/1215

After investigating, I found that this is related to inertial scrolling in Mac, but I find no solution to fix this issue in code.:(

https://github.com/PlayCover/PlayTools/assets/21299239/25926895-3a5e-4bb9-8398-2553f2d43f12

### CODE REVIEW

**Comment by RellikJaeger** [comment]
Can I toggle the entire keymap? The keymap bug also exists in the game chat keyboard typing. It would be better to have a manual toggle shortcut to disable the keymap temporarily during text input typing.

**Comment by AkshayanSingla** [comment]
> Can I toggle the entire keymap? The keymap bug also exists in the game chat keyboard typing. It would be better to have a manual toggle shortcut to disable the keymap temporarily during text input typing.

This is a known bug with smart keymapping. To be able to type in chat, right click genshin -> settings -> disable `Smart Keymapping`

**Comment by RellikJaeger** [comment]
> > Can I toggle the entire keymap? The keymap bug also exists in the game chat keyboard typing. It would be better to have a manual toggle shortcut to disable the keymap temporarily during text input typing.
> 
> This is a known bug with smart keymapping. To be able to type in chat, right click genshin -> settings -> disable `Smart Keymapping`

Thanks for the quick fix! Worked like a charm.

File: PlayTools/Controls/PTFakeTouch/NSObject+Swizzle.m [hunk 1]
```diff
@@ -122,10 +122,6 @@ - (double) get_default_width {
     
 }
 
-- (void) hook_setCurrentSubscription:(VSSubscription *)currentSubscription {
```
Comment (XuYicong) [inline PlayTools/Controls/PTFakeTouch/NSObject+Swizzle.m:125]: Why remove this function

**Review by XuYicong — APPROVED** [review]

**Pull request merged**